deploy/sample/triggers-config.sample.xml

```diff
@@ -0,0 +1,34 @@
+<trgr:triggers xmlns:trgr="http://marklogic.com/xdmp/triggers" xmlns:sec="http://marklogic.com/xdmp/security">
+  <trgr:trigger>
+    <trgr:name>CreateTrigger</trgr:name>
+    <trgr:description>Trigger to enrich XML ingested via MLCP</trgr:description>
+    <trgr:data-event>
+      <trgr:directory-scope>
+        <trgr:uri>/myDir/</trgr:uri>
+        <trgr:depth>1</trgr:depth>
+      </trgr:directory-scope>
+      <trgr:document-content>
+        <trgr:update-kind>modify</trgr:update-kind>
+      </trgr:document-content>
+      <trgr:when>pre-commit</trgr:when>
+    </trgr:data-event>
+    <trgr:module>
+      <trgr:database>${modules-db}</trgr:database>
+      <trgr:root>/triggers/</trgr:root>
+      <trgr:path>log-create.xqy</trgr:path>
+    </trgr:module>
+    <trgr:enabled>true</trgr:enabled>
+    <trgr:permissions>
+      <sec:permission>
+        <sec:capability>read</sec:capability>
+        <sec:role-name>${app-role}</sec:role-name>
+      </sec:permission>,
+      <sec:permission>
+        <sec:capability>update</sec:capability>
+        <sec:role-name>${app-role}</sec:role-name>
+      </sec:permission>
+    </trgr:permissions>
+    <trgr:recursive>false</trgr:recursive>
+    <trgr:task-priority>normal</trgr:task-priority>
+  </trgr:trigger>
+</trgr:triggers>
```

Comment:
Let me give self-testing a bit of thought. Could perhaps be added in between bootstrap/validate of ml7-config and the changed one..